### PR TITLE
# 02 Automating Tasks and # 03 Set up linters

### PR DESCRIPTION
**Pull request by iampikuda**
### Peer Reviewers

Name :  Munachimso Ugorji
Comment : Your voice is not audible enough.

Name : Bolaji Olajide
Comment : Your voice is not audible enough, i can hardly hear you apart from that, i think its cool. You explained and showed working instances of your gulp file which was great, but i don't know if you are expected to explain at least a little to someone that is new to gulp

---

#### What does this PR do?
- This PR serves as my 2nd and 3rd output for checkpoint 1

#### Description of Task to be completed?
- Task enables automatic reload of webpage with change in HTML, CSS and JavaScript files
- Task enables checks by HoundCI on every PR in personal repo using Eslint config file

#### How should this be manually tested?
```
git clone https://github.com/andela-opikuda/Invertedindex.git
cd Invertedindex
npm install
gulp
eslint <file to be tested>
```

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

- Automate reloading [link](https://www.pivotaltracker.com/story/show/140009791)
- Automate code check [link](https://www.pivotaltracker.com/story/show/140009987)

#### Screencast

- [My Screencast](https://www.youtube.com/watch?v=fiv9PO3LmD0 "Visit Damisi's Screencast")

#### Questions:
N/A

### CODE REVIEW

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Setting up repository
+
+## Output
+
+- **Using**: GitHub
+- **Deliverables**: A repository of proposed project and at least one Pull Request
+
+## Link to GitHub Repo
+
+[My InvertedIndex Repo](https://github.com/andela-opikuda/Invertedindex/ "Visit Damisi's Project Repo")
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md]: I still need much more from your ReadMe, kindly consult with your peers.

File: 06-building-a-program-start-to-finish/automating-tasks/output/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- **Deliver**: a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files
+- **With**: *independently*
+
+## Link to GitHub Repo
+
+[My InvertedIndex Repo](https://github.com/andela-opikuda/Invertedindex/ "Visit Damisi's Project Repo")
+
+## Link to Screencast
+
+[My Screencast](https://www.youtube.com/watch?v=fiv9PO3LmD0 "Visit Damisi's Screencast")
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/automating-tasks/output/README.md]: I'd have loved to see you watch for changes in a separate JS file related to your Fronnt-end and not your gulp file but you get the idea. I'll let it pass...

**Review by q-ode — COMMENTED** [review]
The other outputs are fine.

Kindly address the `ReadMe` and let's move this show on...

**Comment by iampikuda** [comment]
All readme.md files revisited and edited

**Comment by rhenshaw56** [comment]
@andela-opikuda hmm, It seems you took my feedback. But just because its your birthday. Good work. I believe you'll still add more to the README as the project progresses.

**Comment by iampikuda** [comment]
@andela-rhenshaw Thanks. I just looked at yours, it seems i was not comparing mine with the right people.
You just remembered didn't you?

**Pull request merged**